html/captive-portal/lib/captiveportal/PacketFence/Controller/Activate/Email.pm

```diff
@@ -217,6 +218,7 @@ sub doSponsorRegistration : Private {
         }
 
         pf::activation::set_status_verified($SPONSOR_ACTIVATION, $code);
+        pf::auth_log::record_completed_guest($activation_record->{source_id}, $node_mac, $pf::auth_log::COMPLETED);
```

Comment:
Sponsor was already working to my knowledge since the record completed is made in the client portal directly

Reply:
I will verify if the sponsor is working as you say.
Then I will just remove that dead code to avoid future confusion.